### ISSUE TITLE
Custom styles example help

### ISSUE THREAD

**Issue by thijselblaso** [mattieha/slider-button-card#86]
Hi, I'm trying to get the custom styles with card-mod working, but nothing is changing. I want to change the action_icon_color. See my config. What am I doing wrong? Can you give me an example config? Thanks

```
cards:
  - type: custom:slider-button-card
    entity: light.keukenlamp
    slider:
      direction: left-right
      background: solid
      use_state_color: false
      toggle_on_click: true
      use_percentage_bg_opacity: false
      force_square: false
      show_track: false
    icon:
      tap_action:
        action: more-info
      use_state_color: false
      icon: ''
      show: true
    action_button:
      mode: custom
      show: true
      icon: ''
      show_spinner: true
      tap_action:
        action: toggle
    compact: false
    show_state: true
    card_mod:
      style: |
        :host {
          --action-icon-color-off: red;
        }
```

**Comment by mattieha** [comment]
@thijselblaso  I added the bug label because it should work with your config.
For now to get it work add  ` !important` to the css line:

```yaml
style: |
  :host {
    --action-icon-color-off: red !important;
  }
```

Also you don't need to add `card_mod:` anymore, although it works both watys.

**Comment by mattieha** [comment]
@thijselblaso This fixed now in v1.10.1 , no need to add the `!important` as mentioned in previous comment.

**Comment by thijselblaso** [comment]
> @thijselblaso This fixed now in v1.10.1 , no need to add the `!important` as mentioned in previous comment.

Thanks! I did a test and almost everything works fine now. Only `--icon-color` still need !important.
